### DISCUSSION TITLE
Android can't be controlled by ios device

### DISCUSSION THREAD

**Discussion by JimmyLX** [rustdesk/rustdesk#5097] [General]
### Bug Description

小米手机安卓被控端无法被 ios/ipad 端控制，只能显示画面
ios 和 ipad 都是 17 的开发者预览版，希望尽快更新苹果端的软件版本，修复这个问题

但同样情况下，用电脑端可以控制安卓，不是权限问题

尝试过重启被控端软件，重启被控手机，还是无法解决

The Android Xiaomi mobile phone cannot be controlled by the ios/ipad device, but it could display the screen
Both ios and ipad are the developer preview version of 17, I hope to update the software version of Apple as soon as possible to fix this problem
But in the same situation, you can control Android with a computer, not a permission issue
Tried to restart the software and the mobile phone, still can not solve the problem

### How to Reproduce

*

### Expected Behavior

*

### Operating system(s) on local side and remote side

Android MIUI14稳定版 -> IOS17 & iPadOS 17开发者预览版

### RustDesk Version(s) on local side and remote side

1.2.1 -> 1.1.10

### Screenshots

*

**Comment by rustdesk** [comment]
@sahilyeole

**Comment by rustdesk** [comment]
double click issue: https://twitter.com/zeroZ53205899/status/1718974477151924245

**Comment by beelchester** [comment]
iphone & ipad (ios17) -> android 12 (simulator & physical). 
Android is controllable from ios. Also no double click issue.

https://github.com/rustdesk/rustdesk/assets/73148455/6e79a128-6704-4c35-aad3-66a74d440653

---
android -> android (sim & physical) controllable and no double click issue.


https://github.com/rustdesk/rustdesk/assets/73148455/60024e94-a0a3-46dc-bc1a-8c5aa3465e7d



Checked nightly and 1.2.1 both

**Reply by rustdesk** [reply]
Thanks

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/5872

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/pull/5880

**Comment by rustdesk** [comment]
require 1.2.3 -> 1.2.3, note, ios 1.2.3 is not avaiable yet, please do not ask me the ETA.

**Comment by zoye999** [comment]
https://github.com/rustdesk/rustdesk/assets/149577366/38924b79-5c82-4511-ad0a-ed287538057a

**Reply by zoye999** [reply]
When an iPhone controls an Android phone，iPhone with a click，The Android phone turns into a double click.What do I have to do to get rid of this bug？
the video : iPhone ios14 and ios17=>Android 12(MIUI12.5.1)，1.2.1-1.2.3 all have the bug of  double click

**Reply by beelchester** [reply]
https://github.com/rustdesk/rustdesk/issues/5872 copy. It has been fixed.
The fix will be available on the next rustdesk ios release.
>require 1.2.3 -> 1.2.3, note, ios 1.2.3 is not avaiable yet, please do not ask me the ETA.
